jsstyle

```diff
@@ -309,6 +309,12 @@ line: while (<$filehandle>) {
 	s/"([^\\"]|\\.)*"/\"\"/g;
 
 	#
+	# Strip out contents of regular expressions so they don't trip us up
```

Comment:
Would you mind updating this comment to mention the cases that we're trying to handle?  Maybe something like: "Strip out contents of easily-identifiable regular expressions so they don't trip us up.  We only attempt to identify those that start with "^" or end with "$", since these are unlikely to be confused with comments, divisions, or other legitimate uses of "/"."  Sorry if this seems nitty, but it took a while for me to grok these regular expressions even knowing what they were supposed to match, and if someone needs to debug them in the future, it would be helpful for them to know what they're supposed to match.